src/promptflow/promptflow/batch/_base_executor_proxy.py

```diff
@@ -65,6 +65,13 @@ def api_endpoint(self) -> str:
         """
         raise NotImplementedError()
 
+    def get_api_path(self, is_aggregation: bool = False) -> str:
```

Comment:
Maybe the parameter could be a string so we are able to extend more apis in the future?